romdisk/uart.asm

```diff
@@ -113,7 +113,9 @@ uartsnd_main:
         dec bc  ; Do not count the command itself
         ld a, b
         or c
+        jp z, _uartsnd_usage
         ret z
```

Comment:
`ret z` will never be taken since there is `jp z` first

Reply:
So how can I modify it? delete `ret z`?

Reply:
Fixed.